INSTALL.md

```diff
@@ -0,0 +1,105 @@
+# Installation
+
+## Table of contents
+
+- [Arch-based distributions](#arch-based-distributions)
+- [Debian](#debian)
+- [Fedora](#fedora)
+- [OpenMandriva](#openmandriva)
+- [openSUSE](#opensuse)
+- [Pop!_OS](#pop_os)
+- [Ubuntu-based distributions](#ubuntu-based-distributions)
+- [AppImage](#appimage)
+- [Flatpak](#flatpak)
+- [Prebuilt releases](#prebuilt-releases)
+- [Source](#source)
+	- [Building](#building)
+	- [Installing](#installing)
+
+## Arch-based distributions
+[`gamehub-git`](https://aur.archlinux.org/packages/gamehub-git/) and [`gamehub`](https://aur.archlinux.org/packages/gamehub/) are available in AUR.
+
+## Debian
+Unfortunately, there is no package available in the Debian repository, so it will be required to run the [`/scripts/build.sh build_deb`](../scripts/build.sh#L171-L210) script. The script will install the dependencies and GameHub.
+
+**Build dependencies**
+* `meson`
+* `valac`
+* `libgtk-3-dev`
+* `libglib2.0-dev`
+* `libwebkit2gtk-4.0-dev`
+* `libjson-glib-dev`
+* `libgee-0.8-dev`
+* `libsoup2.4-dev`
+* `libsqlite3-dev`
+* `libxml2-dev`
+* `libpolkit-gobject-1-dev`
+* `libunity-dev` (optional, required for launcher icon quicklist, progress indicator and counter; pass `-Duse_libunity=true` to `meson` to use)
+* `libmanette-0.2-dev`, `libx11-dev`, `libxtst-dev` (optional, required for gamepad support)
```

Comment:
These are Ubuntu (and probably Debian) packages, but I think dependencies list should be separated and moved to the beginning. Dependencies for other distros are usually named somewhat similarly.

Reply:
sure, i'll get to it ASAP

Reply:
Done https://github.com/tkashkin/GameHub/pull/407/commits/61f18e6bcc00b8fff11fef354f888c814bae904a